x/protocolpool/keeper/keeper.go

```diff
@@ -30,6 +31,19 @@ type Keeper struct {
 	// State
 	Schema         collections.Schema
 	BudgetProposal collections.Map[sdk.AccAddress, types.Budget]
+	ContinuousFund collections.Map[sdk.AccAddress, types.ContinuousFund]
+	// RecipientFundPercentage key: RecipientAddr | value: Percentage in uint64
+	RecipientFundPercentage collections.Map[sdk.AccAddress, uint64]
+	// RecipientFundDistribution key: RecipientAddr | value: Claimable amount
+	RecipientFundDistribution collections.Map[sdk.AccAddress, uint64]
+}
+
+// Global variables
+var toDistribute uint64
```

Comment:
Why is it needed? That looks like a code smell to me.

Reply:
we should write this to state as an item, leaving it global here may have been a miscommunication from me